Geometry/GEMGeometryBuilder/src/GEMGeometryBuilder.cc

```diff
@@ -255,6 +258,7 @@ GEMEtaPartition* GEMGeometryBuilder::buildEtaPartition(DDFilteredView& fv, GEMDe
   pars.emplace_back(ap);
   pars.emplace_back(nStrips);
   pars.emplace_back(nPads);
+  pars.emplace_back(dPhi);
```

Comment:
It looks like this change also needs to be made to `GEMGeometryBuilder::buildEtaPartition(cms::DDFilteredView& fv`.

Reply:
the input arg. is ```DetectorDescription/Core/interface/DDFilteredView.h```
So, the namespace is not ```cms```

Reply:
The function `GEMGeometryBuilder::buildEtaPartition(cms::DDFilteredView& fv, GEMDetId detId)` starting on line 435 of this file needs similar changes to the ones made on lines 230-261.

Reply:
ah, yes, I totally misunderstood. I'm sorry.
I fixed it.
The GEM geometry builder default setting is DD4HEP, so I thought it was working properly with DD4HEP.